### PR TITLE
chore(query): optimize mysql packet read

### LINKED ISSUE #8988: Feature: Split insert values by row batch and insert in parallel
**Summary**

as comment https://github.com/datafuselabs/databend/pull/8966#issuecomment-1328225415

### PR DESCRIPTION
**Pull request by b41sh**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Use the optimized MySQL packet reader [#opensrv 37](https://github.com/datafuselabs/opensrv/pull/37) to improve insert performance. After this PR is merged, the time to insert 10M data will be reduced from 7s to 4s, and the time to insert 20M data will be reduced from 12s to 7s.
A remaining performance problem is that `BlockFilter` and `BlockStatistics` need to be generated when inserting data, waiting for issue #9089 to be fixed.

Closes #8988

### CODE REVIEW

File: src/query/service/Cargo.toml [hunk 1]
```diff
@@ -103,7 +103,7 @@ metrics = "0.20.1"
 naive-cityhash = "0.2.0"
 once_cell = "1.15.0"
 opendal = { version = "0.21", features = ["layers-tracing", "layers-metrics", "compress"] }
-opensrv-mysql = "0.2.0"
+opensrv-mysql = { git = "https://github.com/datafuselabs/opensrv", rev = "5e37788" }
```
Comment (Xuanwo) [inline src/query/service/Cargo.toml:106]: How about bumping a new version for opensrv-mysql?

Comment (b41sh) [reply src/query/service/Cargo.toml:106]: Has discussed this with @PsiACE , and he recommends using rev first.

**Review by Xuanwo — APPROVED** [review]

**Review by BohuTANG — APPROVED** [review]

**Comment by BohuTANG** [comment]
Conflicting files
src/query/service/Cargo.toml

**Comment by Xuanwo** [comment]
It's better to do replace in the top level of `Cargo.toml` to make sure all crates uses the same version.

**Review by PsiACE — APPROVED** [review]

**Pull request merged**